### ISSUE TITLE
nodes created via autoscaling group fail to mount /home and then "instance was taken out of service in response to a user health-check." and repeat ad infinitum 

### ISSUE THREAD

**Issue by PhilipMorrisJones** [aws/aws-parallelcluster#2074]
**Environment:**

aws-parallelcluster-2.7.0

**Bug description and how to reproduce:**

I am not clear on reproduction. I have a running parallelcluster instance with an autoscaling group and Slurm. When a job is submitted the watcher sees no nodes and tells the ASG to add a node and it goes away and provisions a node but that fails so that node is terminated and then the process starts again. 

The problem is that if a compete node fails it wrotes the logs to `/home/logs` but it is failing as it cannot mount `\home` so no log files. 

 ```
- if a compute node was terminated due to failure, there will be a directory 
`/home/logs/compute`. Attach one of the `instance-id.tar.gz` from that directory
```

So it was working

[i-0b895fa88a19fd610.cfn-init.txt](https://github.com/aws/aws-parallelcluster/files/5282819/i-0b895fa88a19fd610.cfn-init.txt)

and then failed

[i-0ac380e114a0c5b5e.cfn-init.txt](https://github.com/aws/aws-parallelcluster/files/5282818/i-0ac380e114a0c5b5e.cfn-init.txt)

**Comment by PhilipMorrisJones** [comment]
I just found out the error. The compute nodes belong to a security group and that needs to be in the security group for the master node. It was removed (in error) so the compute nodes could come up but not access the master node when the mount happens. I assume the mount is the first time the compute node tries to get to the master node.

**Comment by lukeseawalker** [comment]
Yes correct, mount `/home` shared from master [is one of the first action](https://github.com/aws/aws-parallelcluster-cookbook/blob/develop/recipes/_compute_base_config.rb#L57-L65) done by the compute during startup.

Thank you for reporting the resolution of your issue

**Comment by PhilipMorrisJones** [comment]
@lukeseawalker is there any way to add diagnostics for this? It would help greatly if the compute node reported that it could not access the master node.
